xAPI.md

```diff
@@ -509,12 +509,56 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 8.0 Getting More Out of xAPI
```

Comment:
Agreed the title isn't the best, but context is so overloaded already.  We want to add more to xAPI (I was going to say adding definition to xAPI, but definition is also overloaded).  Up for more suggestions!